configure.py

```diff
@@ -196,6 +196,15 @@ def RevolutionLib(lib_name: str, objects: List[Object]) -> Dict[str, Any]:
         "objects": objects,
     }
 
+def LibC(lib_name: str, objects: List[Object]) -> Dict[str, Any]:
+    return {
+        "lib": lib_name,
+        "mw_version": "GC/3.0a3",
+        "cflags": [*cflags_base, "-Cpp_exceptions on", "-O4,p", "-rostr", "-use_lmw_stmw on", "-lang c"],
```

Comment:
it seems a bit weird to have both `-Cpp_exceptions off` and `-Cpp_exceptions on` in the same command line. Maybe instead remove `-Cpp_exceptions off` from `cflags_base`, and add it in the cflags of the other libs?